fiasco/ions.py

```diff
@@ -1473,6 +1539,63 @@ def free_bound(self,
 
         return (prefactor * energy_temperature_factor * sum_factor)
 
+    @u.quantity_input
+    def free_bound_radiative_loss(self) -> u.erg * u.cm**3 / u.s:
+        r"""
+        The radiative loss rate for free-bound emission as a function of temperature,
+        integrated over all wavelengths.
+
+        The calculation integrates Equation 1a of :cite:t:`mewe_freebound_1986`, where the
+        Gaunt factor is summed only for free-bound emission (see CHIANTI technical report #9).
+        Since the form of the Gaunt factor used in by :cite:t:`mewe_freebound_1986` does not
+        depend on wavelength, the integral is straightforward.
+
+        The continuum intensity (per unit EM) is given by:
+
+        .. math::
+
+            P_{fb}(\lambda, T) = \frac{C_{ff} G_{fb}}{\lambda^{2}\ T^{1/2}} \exp{\Big(\frac{-h c}{\lambda k_{B} T}\Big)}
+
+        where :math:`C_{ff} = \frac{64 \pi}{3} \sqrt{\frac{\pi}{6}} \frac{q_{e}^{6}}{c^{2} m_{e}^{2} k_{B}^{1/2}}` is
+        a constant :cite:p:`gronenschild_twophoton_1978`.  Integrating in wavelength space gives the free-bound loss rate:
+
+        .. math::
+
+            R_{fb} = \frac{C_{ff} k_{B} G_{fb} T^{1/2}}{h c} \exp{\Big(\frac{-h c}{\lambda k_{B} T}\Big)}
+
+        We have dropped the factor of :math:`n_{e}^{2}` here to make the loss rate per unit EM.
+
+        Notes
+        -----
+        The input ion `self` is taken to the recombining ion.
+        """
+        z = self.charge_state
+
+        if z == 0:
+            return u.Quantity(np.zeros(self.temperature.shape) * u.erg * u.cm**3 / u.s)
+        else:
+            recombined = self.previous_ion()
+            if not recombined._has_dataset('fblvl'):
+                return u.Quantity(np.zeros(self.temperature.shape) * u.erg * u.cm**3 / u.s)
+
+            C_ff = 64 * np.pi / 3.0 * np.sqrt(np.pi/6.) * (const.e.esu**6)/(const.c**2 * const.m_e**1.5 * const.k_B**0.5)
```

Comment:
Where does the $64/3$ factor come from?

Reply:
This whole function is Equation 1a of [Mewe et al 1986](https://ui.adsabs.harvard.edu/abs/1986A%26AS...65..511M/abstract), but they don't spell out the constants in that paper:

<img width="879" alt="Screenshot 2024-08-09 at 1 04 18 PM" src="https://github.com/user-attachments/assets/a5ec566c-f773-420a-aa8e-09e0fca057a0">
Their earlier paper [Gronenschild & Mewe 1978](https://ui.adsabs.harvard.edu/abs/1978A%26AS...32..283G/abstract) writes out the constants in full (with some unit changes):

<img width="811" alt="Screenshot 2024-08-09 at 1 05 55 PM" src="https://github.com/user-attachments/assets/196aa72b-454b-4f30-9342-30a6a4735b23">

Reply:
Ah I see ok. As long as we have a reference and we've defined the constant as in the paper I think that's fine.

Reply:
I had put the references in the doc string for the function.  If it's not clear, we can always add more detail.